### PR TITLE
Added -N and -C parameters for lymph request

### PR DESCRIPTION
**Pull request by emulbreh**

### CODE REVIEW

File: lymph/cli/request.py [hunk 1]
```diff
@@ -30,47 +31,97 @@ class RequestCommand(Command):
...
+        def timed_request(i):
+            start = time.time()
+            try:
+                request()
+            except Timeout:
+                timeouts.append(i)
+            else:
+                timings.append(1000 * (time.time() - start))
+            request_count = len(timings) + len(timeouts)
+            if request_count % (n / 80) == 0:
```
Comment (mouadino) [inline lymph/cli/request.py]: magic number 80 :) What is this for ? can you replace 80 by a variable that hold some meaning or better wrap it in a function ?

File: lymph/cli/request.py [hunk 2]
```diff
@@ -30,47 +31,97 @@ class RequestCommand(Command):
...
+        n_success = len(timings)
+        n_timeout = len(timeouts)
+        avg = sum(timings) / n_success
+        stddev = math.sqrt(sum((t - avg)**2 for t in timings)) / n_success
+
+        print()
+        print('Requests per second:   %8.2f Hz  (#req=%s)' % (n_success / total_time, n_success))
+        print('Mean time per request: %8.2f ms  (stddev=%.2f)' % (avg, stddev))
+        print('Timeout rate:          %8.2f %%   (#req=%s)' % (100 * n_timeout / float(n), n_timeout))
+        print('Total time:            %8.2f s' % total_time)
```
Comment (mamachanko) [inline lymph/cli/request.py]: `ljust`?

Comment (mamachanko) [inline lymph/cli/request.py]: or a multline template

**Comment by mouadino** [comment]
Can you give an output example ?

File: lymph/cli/request.py (hunk 2 above)
Comment (emulbreh) [inline lymph/cli/request.py]: I felt like ljust was overkill here. A multi line string brings it's own whitespace issues, then we'd have to use textwrap.dedent or something.

Comment (mamachanko) [inline lymph/cli/request.py]: understood. non-blocking of course.

**Comment by emulbreh** [comment]
```
$ lymph request echo.upper '{"text":"foo"}' -N 1000 -C 50
sending 1000 requests, concurrency 50
...................................................................................
Requests per second:     497.69 Hz  (#req=998)
Mean time per request:    81.67 ms  (stddev=1.16)
Timeout rate:              0.20 %   (#req=2)
Total time:                2.01 s

Percentiles:
  0.0 %       3.50 ms (min)
 50.0 %      79.31 ms
 90.0 %     135.49 ms
 95.0 %     148.79 ms
 97.0 %     151.46 ms
 98.0 %     152.20 ms
 99.0 %     153.88 ms
 99.5 %     156.67 ms
 99.9 %     161.81 ms
100.0 %     161.81 ms (max)
```

**Comment by mamachanko** [comment]
don't like the `80` without context either, but +1 otherwise

**Pull request merged**